persistence/testkit/src/main/scala/com/lightbend/lagom/internal/persistence/testkit/PersistenceTestConfig.scala

```diff
@@ -11,52 +11,58 @@ import com.typesafe.config.ConfigFactory
-      "akka.persistence.snapshot-store.plugin"                 -> "cassandra-snapshot-store",
-      "akka.test.single-expect-default"                        -> "5s",
-      "cassandra-journal.contact-points.0"                     -> "127.0.0.1",
-      "cassandra-journal.keyspace"                             -> keyspacePrefix,
-      "cassandra-journal.port"                                 -> cassandraPort.toString,
-      "cassandra-journal.session-provider"                     -> "akka.persistence.cassandra.ConfigSessionProvider",
-      "cassandra-query-journal.eventual-consistency-delay"     -> "2s",
-      "cassandra-snapshot-store.contact-points.0"              -> "127.0.0.1",
-      "cassandra-snapshot-store.keyspace"                      -> keyspacePrefix,
-      "cassandra-snapshot-store.port"                          -> cassandraPort.toString,
-      "cassandra-snapshot-store.session-provider"              -> "akka.persistence.cassandra.ConfigSessionProvider",
-      "lagom.persistence.read-side.cassandra.contact-points.0" -> "127.0.0.1",
-      "lagom.persistence.read-side.cassandra.keyspace"         -> s"${keyspacePrefix}_read",
-      "lagom.persistence.read-side.cassandra.port"             -> cassandraPort.toString,
-      "lagom.persistence.read-side.cassandra.session-provider" -> "akka.persistence.cassandra.ConfigSessionProvider"
-    ) ++ BasicConfigMap
+    ClusterConfigMap ++
+      Map(
+        "akka.persistence.journal.plugin"                        -> "cassandra-journal",
+        "akka.persistence.snapshot-store.plugin"                 -> "cassandra-snapshot-store",
+        "akka.test.single-expect-default"                        -> "5s",
+        "cassandra-journal.contact-points.0"                     -> "127.0.0.1",
+        "cassandra-journal.keyspace"                             -> keyspacePrefix,
+        "cassandra-journal.port"                                 -> cassandraPort.toString,
+        "cassandra-journal.session-provider"                     -> "akka.persistence.cassandra.ConfigSessionProvider",
+        "cassandra-query-journal.eventual-consistency-delay"     -> "2s",
+        "cassandra-snapshot-store.contact-points.0"              -> "127.0.0.1",
+        "cassandra-snapshot-store.keyspace"                      -> keyspacePrefix,
+        "cassandra-snapshot-store.port"                          -> cassandraPort.toString,
+        "cassandra-snapshot-store.session-provider"              -> "akka.persistence.cassandra.ConfigSessionProvider",
+        "lagom.persistence.read-side.cassandra.contact-points.0" -> "127.0.0.1",
+        "lagom.persistence.read-side.cassandra.keyspace"         -> s"${keyspacePrefix}_read",
+        "lagom.persistence.read-side.cassandra.port"             -> cassandraPort.toString,
+        "lagom.persistence.read-side.cassandra.session-provider" -> "akka.persistence.cassandra.ConfigSessionProvider"
+      )
```

Comment:
Cassandra and JDBC always need clustering, so we make its config complete by merging them earlier. 

This code is used in both Java and Scala DSL. This change make it more correct because now we all reuse the same instead of tweaking and composing each time.